packages/manager/tools/testcafe/pages/hub/catalogPage.js

```diff
@@ -0,0 +1,14 @@
+import { Selector, t } from 'testcafe';
+import ManagerParentPage from '../common/managerParent';
+
+export default class CatalogPage extends ManagerParentPage {
```

Comment:
`CatalogPage` isn't used in your tests ?

Reply:
not yet i need some attributes added in this pr to access it properly